omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -129,11 +139,39 @@
 				
 				<prop key="admin/provider/providerAttributeType.list">providerAttributeTypeList</prop>
 				<prop key="admin/provider/providerAttributeType.form">providerAttributeTypeForm</prop>	
-				
+
  			</props>
 		</property>
 	</bean>	
 
+ 	<!-- index pages - begin -->  
+	
+	<!-- admin -->
+	<mvc:view-controller view-name="module/legacyui/admin/index" path="admin/index.htm"/>				
```

Comment:
@wluyima, we have a problem.
I have spent several days on this ticket and this is what seems to be happening.
1st we have to distinguish between a url and a view.
From what I have found a url is processed by a controller to return a view.
The way openmrs (spring) is setup, "/admin" is considered as view, so there is no way to use a controller to map it to a different view eg under "module/legacyui".
To achieve that, we have to use view resolution, which is a bit tricky as it is easy to break the rest of the application.
(openmrs already has a complicated view resolution setup which basically resolves  "/admin" to "WEB-INF/view/admin")
In other words, the parameterizable controllers are not getting a chance to process "/admin".
I am sure this would be the same effect if I wrote a class using "Controller" and "RequestMapping" as that is the code which the paramterizable controller generates anyway.
Do you agree? If so what approach would you recommend.